components/02-elements/swatch/_swatch.scss

```diff
@@ -0,0 +1,79 @@
+$swatch-font-size          : $font-size-small !default;
+$swatch-margin             : 15px !default;
+$swatch-width              : 48px !default;
+$swatch-transition         : $transition-base !default;
+$swatch-background         : $white !default;
+$swatch-label-color        : $gray-light !default;
+$swatch-border-color       : $gray-light !default;
+$swatch-outline-color      : $gray-light !default;
+$swatch-color-hover        : $color-primary !default;
+$swatch-background-active  : $color-primary !default;
+$swatch-border-color-active: $color-primary !default;
+$swatch-color-active       : $white !default;
```

Comment:
You are overusing this variable, you shouldn't use it to set border colors.
And same as you changed `swatch-color` to`swatch-label-color` it should be `swatch-label-color-active`